volestipy/volestipy/include/bindings.h

```diff
@@ -1,31 +1,51 @@
 #ifndef VOLESTIBINDINGS_H
 #define VOLESTIBINDINGS_H
 
-
+//// I think we could remove those...
```

Comment:
which one?

Reply:
I think we could remove the following:
``` 
#include "Eigen/Eigen"
#include "cartesian_kernel.h" 
```
without them, volestipy still compliles and runs ```test1.py``` and ```test2.py```

Fo now, I have just comment them out. Any suggestions? :)

Reply:
probably not needed, you can remove them